enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/AuthProceduresInteractionTestBase.java

```diff
@@ -227,10 +227,10 @@ public void shouldNotCreateUserIfInvalidUsername() throws Exception
                 "The provided username is empty." );
         assertFail( adminSubject, "CALL dbms.security.createUser('', '1234', true)",
                 "The provided username is empty." );
-        assertFail( adminSubject, "CALL dbms.security.createUser('&%ss!', '1234', true)",
-                "Username '&%ss!' contains illegal characters." );
-        assertFail( adminSubject, "CALL dbms.security.createUser('&%ss!', '', true)",
-                "Username '&%ss!' contains illegal characters." );
+        assertFail( adminSubject, "CALL dbms.security.createUser(',ss!', '1234', true)",
+                "Username ',ss!' contains illegal characters." );
+        assertFail( adminSubject, "CALL dbms.security.createUser(':åss!', '', true)",
+                "Username ':åss!' contains illegal characters." );
```

Comment:
I think it slightly better if the usernames here and in the assert above are exactly the same, as the difference is supposed to be in valid vs. invalid password. We test the different illegal characters in the `FileUserRepositoryTest`.